### ISSUE TITLE
[Episode 2] Add markdown cells indicating workflow stage

### ISSUE THREAD

**Issue by steve-crouch** [Southampton-RSG-Training/deep-learning-intro#4]
Add markdown cells at the start of each stage of the workflow, to remind and demarcate where learners are in the overall process. e.g. "5. Choose a loss function and optimizer"

**Comment by steve-crouch** [comment]
Add an instructor note about this before the first step.
